centaur/src/main/resources/standardTestCases/delete_intermediates/no_input_delete_setup.wdl

```diff
@@ -0,0 +1,37 @@
+version 1.0
+
+import "no_input_delete.wdl" as subworkflow
+
+workflow no_input_delete_setup {
+
+    call makeFileAndIndex { }
+    # Pass the result of making the files to the subworkflow to ensure that the makeFileAndIndex task
+    # is complete before the subworkflow runs.
+    call subworkflow.no_input_delete as no_input_delete { input:
+        file_created = makeFileAndIndex.result,
+        f = "gs://cloud-cromwell-dev-self-cleaning/cromwell_execution/no_input_delete/test_execution/no_input_delete.txt"
+    }
+
+    output {
+        String done = makeFileAndIndex.result
+    }
+}
+
+task makeFileAndIndex {
+    meta {
+        volatile: true
+    }
+    String outputFile = 'output.txt'
+    input {
+    }
+    command {
+        gsutil cp gs://cloud-cromwell-dev-self-cleaning/cromwell_execution/no_input_delete/source_files/*.* gs://cloud-cromwell-dev-self-cleaning/cromwell_execution/no_input_delete/test_execution
```

Comment:
Note that this requires the source_files directory to have the appropriate data. We copy it over to test_execution so that if a previous test case failed due to the file being incorrectly deleted, the next run does not fail. But of course all test runs are hitting the same directory, so it's possible that a failing test could impact other test runs happening at "the same time".

![image](https://user-images.githubusercontent.com/484484/135677151-c92e15e8-b79c-4dbe-b2ab-0ba4b7a3a18b.png)